From 408b19227bbccf316e91bb5c8f7732b3a9838ac6 Mon Sep 17 00:00:00 2001
From: "Luke D. Jones" <luke@ljones.dev>
Date: Wed, 6 Nov 2024 00:27:03 +0300
Subject: [PATCH 093/188] debug by default

(cherry picked from commit 8be2ebf58bde7ad79a2d42ca8730667867ba1499)
---
 drivers/hid/hid-asus-ally.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/hid/hid-asus-ally.c b/drivers/hid/hid-asus-ally.c
index e6ff4ba08..e78625f70 100644
--- a/drivers/hid/hid-asus-ally.c
+++ b/drivers/hid/hid-asus-ally.c
@@ -22,6 +22,8 @@
 #include "hid-asus.h"
 #include "hid-asus-ally.h"
 
+#define DEBUG
+
 #define READY_MAX_TRIES 3
 #define FEATURE_REPORT_ID 0x0d
 #define FEATURE_ROG_ALLY_REPORT_ID 0x5a
-- 
2.55.0

